www/src/components/feedback-widget/lazy-feedback-widget.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react"
+import { ToggleButton, OpenFeedbackWidgetButtonContent } from "./buttons"
+import { WidgetContainer } from "./styled-elements"
+
+const LazyFeedbackWidget = () => {
+  // We need to use wrapping object
+  // setWidget(<func>) would execute the function
+  // and we don't want that for function components
+  const [{ Widget, loading }, setWidget] = useState({
+    Widget: null,
+    loading: false,
+  })
+
+  const [wasClicked, setWasClicked] = useState(false)
+  const [wasFocused, setWasFocused] = useState(false)
+
+  const handleToggle = () => {
+    setWasClicked(true)
+    setTimeout(() => {
+      // if we didn't load widget within 1 second, show loading indicator
+      setWidget(state => {
+        if (!state.Widget) {
+          return {
+            loading: true,
+          }
+        }
+        return state
+      })
+    }, 1000)
```

Comment:
following rail practices https://developers.google.com/web/fundamentals/performance/rail this should be 100ms.

Reply:
I have no strong opinion on this (but I think 100ms might be to short?) - I've used timeout we use already in core https://github.com/gatsbyjs/gatsby/blob/f4e8d618e4cc77624df9b3d1779b2dcac2fdfcc5/packages/gatsby/cache-dir/navigation.js#L75-L82
for plugins like `gatsby-plugin-nprogress` to show loader on page transitions